config/sync/core.entity_view_display.rdf_entity.solution.group_header.yml

```diff
@@ -89,6 +89,19 @@ content:
       template_suggestion:
         template_suggestion: ''
     region: content
+  field_policy_domain:
+    type: entity_reference_custom_link
+    weight: 4
+    region: content
+    label: inline
+    settings:
+      path: 'internal:/search'
+      label: '[term:name]'
+      query_parameters: "f[0]|policy_domain:[term:tid]\r\nf[1]|type:solution"
+      limit: '7'
```

Comment:
The `limit` should be an integer. Then, why it was saved as a string? Simply because your formatter settings are missing a schema. This is the reason why this hack [`c1b8b52` (#2263)](https://github.com/ec-europa/joinup-dev/pull/2263/commits/c1b8b524d7b6b30e735a8d04af8052d7b83c8434) was necessary. In fact that was only hiding the real problem.